examples/hybrid/sphere/single_column_radiative_equilibrium.jl

```diff
@@ -12,17 +12,19 @@ params = EarthParameterSet()
 horizontal_mesh =
     periodic_rectangle_mesh(; x_max = Δx, y_max = Δx, x_elem = 1, y_elem = 1)
 quad = Spaces.Quadratures.GL{1}()
-z_max = FT(100e3)
-z_elem = 100
-t_end = FT(60 * 60 * 24 * 365.25 * 5)
+z_max = FT(90e3)
+z_elem = 90
+z_stretch = Meshes.GeneralizedExponentialStretching(FT(100), FT(10000))
+t_end = FT(60 * 60 * 24 * 365.25)
```

Comment:
Can we use the cli option here instead?

Reply:
Hmm, passing something like `Meshes.GeneralizedExponentialStretching(FT(100), FT(10000))` through a CL argument can get rather unwieldy. I agree that we should be able to do it, but maybe we can do that in a separate PR. Also, it would be nice for examples to come with "default meshes", so that I can just run the radiative equilibrium example from the REPL without worrying about starting Julia with a particular `--stretch` argument.

Reply:
I'm fine with merging for now, but this is an increasing problem since this script cannot be included with any other `TEST_NAME` script since each overwrite different global variables.

We can develop a more unified solution to making a generic script repl-friendly.

In the long term, much of this code is going to need to move into `ClimaAtmos.jl/src`, and increasing the number of global variables makes that more difficult.

Reply:
Yes, I definitely agree that all of our driver-setup-related global variables need to be encapsulated in a data structure, and an order of precedence should be established for declaring these variables in different places (e.g., declaring a default `dt` for all baroclinic wave examples, declaring a default `dt` for a particular example, declaring a specific `dt` through a command-line argument, and declaring a specific `dt` through the REPL). I think I will work on this starting next Monday.